api/components/BlobUpload.tsx

```diff
@@ -49,7 +49,7 @@ const uploadFiles = async (acceptedFilesList: ParsedFile[], plot: string, userIn
             date: (new Date()).toDateString(),
           },
           tags: {
-            uploadedBy: userInfo.userDetails,
+            plot: plot,
```

Comment:
Instead of removing `uploadedBy`, consider sanitizing the user details so it can be accepted in this tag.

Reply:
I left the plot tag, added user back with replacing the '@' character.